vcpkg/concepts/dependabot.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Dependabot support for vcpkg
+description: This article describes how to use GitHub Dependabot to automatically update vcpkg dependencies in your projects.
+author: jamagee
+ms.author: jamagee
+ms.date: 8/5/2025
+ms.topic: concept-article
+---
+# Dependabot support for vcpkg
+
+[Dependabot](https://docs.github.com/en/code-security/dependabot) is GitHub's automated dependency management service
+that helps keep your project dependencies up-to-date by automatically creating pull requests when new versions are
+available. Starting in 2025, Dependabot now supports [vcpkg](https://vcpkg.io/).
+
+## How Dependabot works with vcpkg
+
+When you enable Dependabot for your vcpkg projects, it monitors your `vcpkg.json` manifest files and automatically
+creates pull requests to update the `builtin-baseline` commit hash. This ensures your C/C++ dependencies stay current
+with the latest versions available in the vcpkg port repository.
+
+## Setting up Dependabot for vcpkg
+
+To enable Dependabot version updates for your vcpkg project:
+
+1. Create or update your `.github/dependabot.yml` file in your repository's default branch:
+
+```yaml
+version: 2
+updates:
+  - package-ecosystem: "vcpkg"
+    directory: "/" # Location of your vcpkg.json file
+    schedule:
+      interval: "weekly"
+```
+
+1. Customize the configuration based on your project needs:
+
+```yaml
+version: 2
+updates:
+  - package-ecosystem: "vcpkg"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "09:00"
+      timezone: "America/New_York"
+    labels:
+      - "dependencies"
+      - "vcpkg"
+    commit-message:
+      prefix: "deps"
+      include: "scope"
+    open-pull-requests-limit: 5
+```
+
+## Configuration options
+
+Dependabot for vcpkg supports all standard Dependabot configuration options. The key vcpkg-specific requirements are:
+
+- `package-ecosystem` must be set to `"vcpkg"`
+- `directory` should point to the location of your `vcpkg.json` file
+
+For comprehensive configuration options see the [Dependabot options reference][dependabot-options].
+
+### vcpkg-specific considerations
+
+- **Test thoroughly**: Baseline updates can introduce breaking changes in dependencies.
+  Always test your build after applying updates.
+- **Monitor vcpkg announcements**: Watch the [vcpkg repository][vcpkg-repo] for announcements
+  about breaking changes or important updates.
+- **Consider version pinning**: If you need stability, consider pinning specific dependency
+  versions in your `vcpkg.json` rather than relying solely on baseline updates.
+
+### vcpkg-specific issues
+
+**Dependabot not creating pull requests:**
+
+- Ensure your repository has a valid `vcpkg.json` with a `builtin-baseline` field
+- Verify the `directory` path in your configuration points to the correct location
+
+**Build failures after baseline updates:**
+
+- Baseline updates may introduce breaking changes in dependencies
```

Comment:
```suggestion
- Baseline updates may introduce breaking changes from dependencies
```
Also, adding a comment from my colleague @vicroms: does this refer to introducing breakage due to version conflicts? Or because a baseline update may introduce a "major" version change?

Reply:
Definitely the latter.